resources/clojure.land/projects.edn

```diff
@@ -5958,4 +5958,22 @@
   :url "https://github.com/kkinnear/zprint",
   :platforms #{"clj" "cljs"},
   :key :zprint,
-  :tags #{"Code Formatting Tools"}}]
+  :tags #{"Code Formatting Tools"}}
+ {:name "config",
```

Comment:
I'm trying to keep the project list in alphabetical order by name/key.  Please move these to their appropriate places.